ulib/LowStar.ConstBuffer.fsti

```diff
@@ -0,0 +1,187 @@
+(*
+   Copyright 2008-2019 Microsoft Research
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*)
+module LowStar.ConstBuffer
+
+(* This module provides a model of const pointers in C.
+
+   A well-typed module guarantees that it will not mutate memory
+   through a const pointer. But it cannot rely on the context not
+   mutating the same memory.
+
+   As such, we model const pointers as a finite disjunction of
+   {mutable, immutable}-pointers, forcing code to guarantee the
+   strongest condition of the two (immutability) and to rely only on
+   the weakest (i.e., mutability).
+
+   The main type of this module is `const_buffer t`. It is extracted
+   by KreMLin to  `const t*`, i.e., a
+*)
+
+module P = FStar.Preorder
+module G = FStar.Ghost
```

Comment:
As far as I can see abbreviations `P` and `G` are not used anywhere (in both the files).

Reply:
good call. thanks, i'll remove